themes/10up-theme/assets/css/frontend/02-global/reset.css

```diff
@@ -0,0 +1,83 @@
+/**
+* Reset.css
+* Custom reset enhancments.
+* Primary reset is modern-normalize (by default)
+* imported in ../style.css
+*/
+
+/*
+ * Reset spacing & font inheritance
+ *
+ * Font iheriance: We add intentional font inheritence as there are browser
+ * inconstencies as well as some instances where lacking implecit inheritence
+ * can cause challenges
+ *
+ * Margin & Padding reset: We reset margin and padding to 0 to ensure a
+ * consistentstarting point for all elements
+ */
+* {
+	font: inherit;
+	margin: 0;
+	padding: 0;
+}
+
+html {
+
+	/* enable dark and light color schemes by default */
+	color-scheme: dark light; /* stylelint-disable-line scale-unlimited/declaration-strict-value */
```

Comment:
Do we want to enable dark mode by default? We don't usually style for it unless it is a specific requests. 

Also I don't think we should have to override our own linting rules in our scaffold.

Reply:
I don't think this enables dark mode by default, just that it allows it to be both dark and light